### ISSUE TITLE
Suggestion: PLAYER_MODEL Overlay should still show green instead of yellow at fractions of 1 damage taken per limb

### ISSUE THREAD

**Issue by StygianEmperor** [ichttt/FirstAid#109]
**1.16.4-forge-35.1.29**
**firstaid-1.9.5**

The PLAYER_MODEL Overlay is currently too strict for what amount of health it considers "in the green". Even if it appears that I have full hearts in a limb, it will still show that limb as yellow (AKA hurt).

This is because, apparently, Minecraft considers something around 0.9 damage (?, didn't test thoroughly) to not be enough damage to keep from displaying a full heart, whereas the PLAYER_MODEL Overlay seemingly considers even 0.1 damage to be enough to stop showing green on any given limb.

**TL;DR**
Essentially, the PLAYER_MODEL Overlay appears to be giving inaccurate reports of limb HP (even though it technically isn't - though 0.9 damage usually isn't enough to worry me with a non-green display), and should probably be synchronized to the same threshold for when Minecraft decides to display missing hearts. I hope that was clear.

**Edit:** actually, configurable colors and thresholds in general would probably be cool.

**Comment by ichttt** [comment]
Somewhat fixed in 09c357a019dc2dc7cf5ff255a22db2d34838a99b, at least there is a wider range of colors now
